vtm-ios/src/org/oscim/ios/backend/IosGraphics.java

```diff
@@ -20,41 +44,39 @@ public static void init() {
 		g = INSTANCE;
 	}
```

Comment:
See my latest commit 56dd66e1f8d76c2f6c5e64847b579dd4e53024bf in `issue_28' branch.
We can initialize `IosGraphics` the way other platforms do.
That way the `CanvasAdapter` needs no modifications.

Also is the `IosGraphics.get` needed?